### PR TITLE
People and Appointment Search

### PR DESCRIPTION
**Pull request by mghall25**
Fixes appointment search so it works. Adds the ability to search on people.  Applies to issue #44

### CODE REVIEW

File: shopyo/templates/people/lookup.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            <th></th>
+        </tr>
+        </thead>
+        <tbody>
+            {% for person in people %}
+            <tr class="{% if person.active == "inactive" %}all_not{% endif %}">
+                <td>{{ person.name }}</td>
+                <td>{{ person.date }}</td>
+                <td>{{ person.time }}</td>
+                <!--<td><a href="/settings/edit/{{person.name}}" class="btn btn-info" role="button"><i class="fas fa-pencil-alt"></i></a></td>-->
```
[the remark this answers is not in the document]
Comment (mghall25) [reply shopyo/templates/people/lookup.html]: @arthurarty Thanks! Done.

[the remark this answers is not in the document]
Comment (arthurarty) [reply shopyo/templates/people/lookup.html]: You welcome.

**Comment by Abdur-rahmaanJ** [comment]
Thank you @mghall25 🍃

**Comment by Abdur-rahmaanJ** [comment]
But there is  something off:

in people/lookup.html

```

  <div id="search_results">
  </div>
  <br><br>
  <table class="table" id="results_table">
        <thead>
        <tr>
            <th scope="col">Meeting Name</th>
            <th scope="col">Date</th>
            <th scope="col">Time</th>
            <th scope="col">Active</th>
            <th></th>
        </tr>
        </thead>
```

Needs to fix fields name

**Comment by mghall25** [comment]
Fixed - pull request #77.

**Pull request merged**